cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py

```diff
@@ -153,7 +152,11 @@ def execute_command_on_server(self):
         custom_values = {"log": {"label": log_label}}
         for server in self.server_ids:
             server.execute_command(
-                self.command_id, sudo=self.use_sudo, path=self.path, **custom_values
+                self.command_id,
+                sudo=self.use_sudo,
+                path=self.env.user.has_group("cetmix_tower_server.group_manager")
```

Comment:
`path=self.env.user ......` you can take it out of the loop so that you don't have to calculate this data every time.